orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/MonitorKatoTask.groovy

```diff
@@ -108,8 +108,8 @@ class MonitorKatoTask implements RetryableTask, CloudProviderAware {
         katoTask = kato.lookupTask(taskId.id, false)
       }, MAX_HTTP_INTERNAL_RETRIES, Duration.ofMillis(100), false)
       outputs['kato.task.notFoundRetryCount'] = 0
-    } catch (RetrofitError re) {
-      if (re.kind == RetrofitError.Kind.HTTP && re.response.status == HttpURLConnection.HTTP_NOT_FOUND) {
+    } catch (SpinnakerHttpException re) {
+      if (re.getResponseCode() == HttpURLConnection.HTTP_NOT_FOUND) {
```

Comment:
There's a bit of mix and matching going on between magic numbers and enums. Does it make sense to settle on one approach?

Reply:
It does.  It's a level of hygiene I wasn't focused on while doing this unfortunately.  Madly trying to get rid of retrofit1.